src/AutoComplete/AutoComplete.component.tsx

```diff
@@ -16,6 +16,7 @@ type AutoCompleteDataSource = Array<{
 }>;
 
 interface AutoCompleteProps {
+    inputComponent?: any;
```

Comment:
Following the pattern of other Anchor components that accept components as a prop. `ReactElement` throws TS errors, I'm guessing because of the _forwardRef_?

Reply:
Probably, why not use the `as` prop built into styled-components? I think I'm not up to speed on the ask here, though. Wanna chat about that in person or dump it here?